Content.Server/Administration/Systems/BwoinkSystem.cs

```diff
@@ -80,6 +84,22 @@ public override void Initialize()
 
             SubscribeLocalEvent<GameRunLevelChangedEvent>(OnGameRunLevelChanged);
             SubscribeNetworkEvent<BwoinkClientTypingUpdated>(OnClientTypingUpdated);
+
+            _rateLimit.Register(
+                RateLimitKey,
+                new RateLimitRegistration
+                {
+                    CVarLimitPeriodLength = CCVars.AhelpRateLimitPeriod,
+                    CVarLimitCount = CCVars.AhelpRateLimitCount,
```

Comment:
Maybe add an UpdateRegistration method or something, so these can change with the cvars on a running server?

Reply:
The rate limiting system subscribes to the CVar changes itself, that's why it takes CVars instead of just the values directly.

Reply:
Ah, I missed that. Awesome!